client/rest-high-level/src/main/java/org/elasticsearch/client/IndexLifecycleClient.java

```diff
@@ -300,4 +306,120 @@ public void retryLifecyclePolicyAsync(RetryLifecyclePolicyRequest request, Reque
         restHighLevelClient.performRequestAsyncAndParseEntity(request, IndexLifecycleRequestConverters::retryLifecycle, options,
             AcknowledgedResponse::fromXContent, listener, emptySet());
     }
+
+    /**
+     * Retrieve one or more snapshot lifecycle policy definitions. See
+     * <a href="fill-me-in">
```

Comment:
I think we can fill in these links now that the asciidoc is written. It won't be on the website obviously, but it's pretty straightforward to determine the URLs.

I'm okay with leaving this if there's something I'm missing, though.

Reply:
Thanks Gordon, I filled in the links.